pages/personal.tsx

```diff
@@ -1,38 +1,81 @@
 import dynamic from 'next/dynamic'
-import ProfileImage from '../components/ProfileImage'
+import Head from 'next/head'
 import { NavBar } from '../components/NavBar'
+import ProfileImage from '../components/ProfileImage'
 import { PersonalDescription } from '../components/PersonalDescription'
-import { Container, Center } from '@chakra-ui/react'
-import { useState } from 'react'
-import { CommentSection } from '../components/CommentSection'
+import { Box, SimpleGrid, VStack } from '@chakra-ui/react'
+import { useState, useEffect } from 'react'
 
 const LazyVisualizer = dynamic(() => import('../components/AudioVisualizer'), {
   ssr: false
 })
 
 const personal = () => {
-  interface Data {
-    imgURL: string,
-    name: string,
-    instruments: string[],
-  }
+
+  useEffect(() => {
+    fetch('api/userFeed')
+      .then(async (response) => {
+        const newData = await response.json()
+        setData(newData)
+      })
+  },[])
 
   const [data, setData] = useState({
-    imgURL: 'https://newprofilepic2.photo-cdn.net//assets/images/article/profile.jpg',
-    name: 'Tracy Hillberg',
-    instruments: ['cello', 'flute', 'drums'],
-  })
+    "name": "Ivan",
+    "description": "hello world",
+    "instruments": [
+        "Cello",
+        "Piano",
+        "Drums"
+    ],
+    "image": "testprofilepicture.jpg",
+    "posts": [
+        {
+            "name": "Joe",
+            "band": "Super Sick Band",
+            "audio": "testaudio.wav",
+            "pdf": "testpdf.pdf",
+            "date": "01/17/2023 @ 8:09pm",
+            "text": "Hello user feed",
+            "comments": [
+                {
+                    "name": "Darrien",
+                    "profile_picture": "sampleprofpic.jpg",
+                    "text": "hello comments",
+                    "date": "01/17/2023 @ 8:10pm"
+                },
+                {
+                    "name": 'Joe',
+                    "profile_picture": 'pfp.jpeg',
+                    "text": "test",
+                    "date": '01/17/2023 @ 8:11pm'
+                }
+            ]
+        }
+    ]
+})
 
 
   return(
     <>
-      <NavBar />
-      <Container>
-        <ProfileImage imgURL={data.imgURL} name={data.name}/>
-        <PersonalDescription instruments={data.instruments} sectionName='personal'/>
-        <LazyVisualizer/>
-        <CommentSection/>
-      </Container>
+    <Head>
+      <title>Your Homepage</title>
+    </Head>
+
+      <Box h='100vh' w='100vw'>
+        <NavBar/>
+        <Box display='flex'>
+            <SimpleGrid columns={2} spacing={5} alignContent='center'>
+              <VStack>
+                <ProfileImage image={data.image} name={data.name}/>
+                <PersonalDescription description={data.description} instruments={data.instruments}/>
+              </VStack>
+              <VStack>
+                <LazyVisualizer posts={data.posts}/>
```

Comment:
I think we may have an issue with using wavesurfer. Since we are providing it with element IDs to select the right HTML components, some errors occur if two visualizers are loaded onto one page. If you add a second `<LazyVisualizer posts={data.posts}/>` below this one to render two posts, only the first post works as intended. Do you have anything in mind to deal with this?